docs/en/latest/tutorials/cache-api-responses.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Cache API responses
+keywords:
+  - API Gateway
+  - Apache APISIX
+  - Cache
+  - Permormance
```

Comment:
```suggestion
  - Performance
```

Reply:
Yeah, I run Grammarly actually. In my VS code when typed, did not notice it and made another push. Good point!